modules/infra/datapath/slow_path_input.c

```diff
@@ -0,0 +1,123 @@
+// SPDX-License-Identifier: BSD-3-Clause
+// Copyright (c) 2024 Christophe Fontaine
+
+#include "gr_slow_path.h"
+
+#include <gr_eth_input.h>
+#include <gr_eth_output.h>
+#include <gr_graph.h>
+#include <gr_log.h>
+#include <gr_mbuf.h>
+
+#include <rte_ether.h>
+#include <rte_graph_worker.h>
+#include <rte_malloc.h>
+#include <rte_version.h>
+
+enum {
+	UNKNOWN_SLOW_PATH_TYPE,
+	EDGE_COUNT,
+};
+
+struct gr_slow_path_msg {
+	uint16_t type;
+	void *data;
+};
+
+static struct rte_ring *slow_path_ring;
+static struct rte_mempool *slow_path_pool;
+
+static rte_edge_t slow_path_edges[1 << 16] = {UNKNOWN_SLOW_PATH_TYPE};
```

Comment:
You can make this array a lot smaller if slow path type is uint8_t.

Reply:
fixed.